### PR TITLE
Add RejectUnknownRevocationStatus stack option

### PR DESCRIPTION
**Pull request by cristipogacean**
* Add configuration option for stack's security option RejectUnknownRevocationStatus in edge modules
* Addressing  #1771

### CODE REVIEW

File: docs/modules/discovery.md [hunk 1]
```diff
@@ -203,6 +203,13 @@ The following list of module parametrization settings can be provided either as
                                       Type: boolean
                                       Default: true
 
+            RejectUnknownRevocationStatus=VALUE
+                                      OPC UA Client Security Config - aReject chain validation
```
Comment (karok2m) [inline docs/modules/discovery.md]: `aReject` does not look like a valid word. Maybe the `a` in the font should be removed ?

File: docs/modules/publisher-commandline.md [hunk 2]
```diff
@@ -379,6 +379,14 @@ The `Alternative` field, where present, refers to the CLI argument applicable in
                                       Mode: Standalone and Orchestrated
                                       Type: boolean
                                       Default: true
+            RejectUnknownRevocationStatus=VALUE
+                                      OPC UA Client Security Config - aReject chain validation
```
Comment (karok2m) [inline docs/modules/publisher-commandline.md]: `aReject` does not look like a valid word. Maybe the `a` in the font should be removed ?

File: docs/modules/twin.md [hunk 3]
```diff
@@ -172,6 +172,13 @@ The following list of module parametrization settings can be provided either as
                                       Type: boolean
                                       Default: true
 
+            RejectUnknownRevocationStatus=VALUE
+                                      OPC UA Client Security Config - aReject chain validation
```
Comment (karok2m) [inline docs/modules/twin.md]: `aReject` does not look like a valid word. Maybe the `a` in the font should be removed ?

File: docs/modules/discovery.md [hunk 4]
```diff
@@ -203,6 +203,13 @@ The following list of module parametrization settings can be provided either as
                                       Type: boolean
                                       Default: true
 
+            RejectUnknownRevocationStatus=VALUE
+                                      OPC UA Client Security Config - aReject chain validation 
+                                      with CA certs with unknown revocation status, e.g.when the
```
Comment (karok2m) [inline docs/modules/discovery.md]: There should be a space between `e.g.` and `when` in `e.g.when`.

File: docs/modules/publisher-commandline.md [hunk 5]
```diff
@@ -379,6 +379,14 @@ The `Alternative` field, where present, refers to the CLI argument applicable in
                                       Mode: Standalone and Orchestrated
                                       Type: boolean
                                       Default: true
+            RejectUnknownRevocationStatus=VALUE
+                                      OPC UA Client Security Config - aReject chain validation 
+                                      with CA certs with unknown revocation status, e.g.when the
```
Comment (karok2m) [inline docs/modules/publisher-commandline.md]: There should be a space between `e.g.` and `when` in `e.g.when`.

File: docs/modules/twin.md [hunk 6]
```diff
@@ -172,6 +172,13 @@ The following list of module parametrization settings can be provided either as
                                       Type: boolean
                                       Default: true
 
+            RejectUnknownRevocationStatus=VALUE
+                                      OPC UA Client Security Config - aReject chain validation 
+                                      with CA certs with unknown revocation status, e.g.when the
```
Comment (karok2m) [inline docs/modules/twin.md]: There should be a space between `e.g.` and `when` in `e.g.when`.

**Review by karok2m — COMMENTED** [review]
@cristipogacean, please fix minor issues in the docs. Everything else looks good.

File: docs/modules/discovery.md (hunk 1 above)
Comment (cristipogacean) [reply docs/modules/discovery.md]: done

File: docs/modules/publisher-commandline.md (hunk 2 above)
Comment (cristipogacean) [reply docs/modules/publisher-commandline.md]: done

File: docs/modules/twin.md (hunk 3 above)
Comment (cristipogacean) [reply docs/modules/twin.md]: done

File: docs/modules/discovery.md (hunk 4 above)
Comment (cristipogacean) [reply docs/modules/discovery.md]: done

File: docs/modules/publisher-commandline.md (hunk 5 above)
Comment (cristipogacean) [reply docs/modules/publisher-commandline.md]: done

File: docs/modules/twin.md (hunk 6 above)
Comment (cristipogacean) [reply docs/modules/twin.md]: done

**Review by mregen — APPROVED** [review]
:shipit:

**Review by karok2m — APPROVED** [review]

**Pull request merged**